src/java.base/share/classes/java/net/URL.java

```diff
@@ -440,7 +436,7 @@ public URL(String protocol, String host, int port, String file,
             }
         }
 
-        protocol = toLowerCase(protocol);
+        protocol = URLUtil.lowerCaseProtocol(protocol);
```

Comment:
Maybe we could keep the name 'toLowerCase' in this file

Reply:
I prefer explicitly calling out protocol since it's not intended to be used as a general purpose lower case utility method.